content/applications/finance/fiscal_localizations/united_states.rst

```diff
@@ -0,0 +1,746 @@
+    These appear towards the end of the payslip before the net amount is displayed.
+
+.. seealso::
+   :doc:`Employees documentation <../../hr/employees/new_employee>`
+
+.. _l10n_us/adp:
+
+Export work entries to ADP
+--------------------------
+
+Requirements
+~~~~~~~~~~~~
+
+In order to create a report that can be uploaded to ADP, there are some configurations that must be
+done.
+
+First, ensure the *United States - Payroll - Export to ADP* (`l10n_us_hr_payroll_adp`) module is
+:ref:`installed <general/install>`.
+
+Then, the company **must** have an *ADP Code* entered for the company settings. To do so, navigate
+to :menuselection:`Payroll app --> Configuration --> Settings`. Enter the :guilabel:`ADP Code` in
+the :guilabel:`US Localization` section.
+
+Next, work entry types **must** have the correct ADP code listed in the *External Code* field for
+each work entry type that is being referenced.
+
+Lastly, every employee **must** have an *ADP Code* entered on their employee form. To do so,
+navigate to :menuselection:`Employees app`, select an employee record, and open the :guilabel:`HR
+Settings` tab. Enter the :guilabel:`ADP Code` in the :guilabel:`ADP Information` section.
+
+The :guilabel:`ADP Code` code is how ADP identifies that particular employee, and is typically a
+six-digit number.
+
+.. seealso::
+   - :ref:`payroll/new-work-entry`
+   - :doc:`../../hr/employees/new_employee`
+
+Export data
+~~~~~~~~~~~
+
+Once :doc:`work entries <../../hr/payroll/work_entries>` have been verified, the information can be
+exported into a CSV file, which can then be uploaded into ADP.
+
+To export the data, navigate to :menuselection:`Payroll app --> Reporting --> United States: ADP
+Export`, then click :guilabel:`New`. Next, enter the :guilabel:`Start Date` and :guilabel:`End Date`
+for the work entries using the calendar pop-over.
+
+Then, enter a :guilabel:`Batch ID` in the corresponding field. The recommendation for this field is
+to enter the date in a `YY-MM-DD` format, followed by any other characters to distinguish that
+specific batch, such as a department name, or any other defining characteristics for the batch.
+
+Enter a :guilabel:`Batch Description` in the corresponding field. This should be short and
+descriptive, but distinct from the :guilabel:`Batch Name`.
+
+Ensure the correct company populates the :guilabel:`Company` field. Change the selected company with
+the drop-down menu, if needed.
+
+Lastly, add the employee's work entry information to the list. Click :guilabel:`Add a line` and an
+:guilabel:`Add: Employee` pop-up window loads. The list can be :doc:`filtered
+<../../essentials/search>` to more easily find the employees to add to the list.
+
+.. tip::
+   It is helpful to process the data export in multiple groups instead of in one large group
+   containing all employees. The most common ways to group employees is by department or by wage
+   type (hourly or salaried).
```

Comment:
Context
```suggestion
   Process the data export in multiple groups instead of in one large group that contains all
   employees. This helps to meaningfully differentiate the batches and makes processing more
   tenable, overall. The most common ways to group employees is by department, or by wage type
   (hourly or salaried).
```